src/content/writing/2019-10-09-on-goals-and-happiness.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: 'On Goals and Happiness'
+published: '2019-10-09'
+tags: ['existing in the world']
+preview: [
+  "For as long as I have had a recognizable sense of identity,",
+  "my goal in life has been to be happy.",
+  "This is an unworkable goal."
+]
+---
+
+import { Box, Text, Link } from 'rebass'
+import { Footnote } from '../../components'
+
+<Box fontSize={3}>
+
+#### A Sentence
+
+Happiness is not a goal; it is an outcome of engaging with both the positive and negative aspects of my life.
+
+#### A Paragraph
+
+For as long as I have had a recognizable sense of identity, my goal in life has been to _be happy_. This is an unworkable goal, and I used it for a long time as a defense mechanism to avoid necessary conflict. In order to stay _happy_, I intellectualized negative feelings to the point that they were an ignorable background noise in my brain -- until they were no longer ignorable. Eventually I learned that happiness is a path that must be walked actively. Many steps come only after engaging with, understanding, and resolving negative feelings by stepping into my life. It is useful, and it is difficult, work.
+
+#### A Page
+
+The particular idea of who I am goes back only ten or twelve years. Any further past and I have wide impressions: an easy ten-year-old desire to be, so absurdly generally, a _scientist_; the sharp sense of freedom while driving alone late at night with a new license<Footnote id="foot-1" href="#1">1</Footnote>. I can see the beginnings of myself, but the patterns and systems of my brain, my sense of being, do not feel coherently connected to who I am now. At eighteen I picked up the first thread of my personality that I can follow through to today -- when asked what I wanted to do, or to be, I would say that I only wanted to be happy. That was my goal; I am not goal oriented.
+
+I carried this goal, _happiness_, through school<Footnote id="foot-2" href="#2">2</Footnote> and along through the beginnings of my adulthood and career. I'll say at the outset, here, that happiness is a good direction and an awful goal. I'll say also that I was more earnest than smugly anti-establishment; I inherited a certain casual _everything works out_ attitude from my mom that I intellectualized to the point of being deeply unhelpful. It's true, too, that everything does work out, one way or another, and that I've been lucky. I come from a background of white- and male-privilege. I'm grateful for what I have -- what I've been given and what I've earned.
```

Comment:
Whenever me and my sisters would complain about something being fair or unfair, my mom would always tell us that "everything evens out when you're 21."